chores/views.py

```diff
@@ -99,6 +99,7 @@ def index(request):
     context = {
         "title": "Chores",
         "event_groups": event_groups,
+        "has_permission": request.user.is_authenticated,
```

Comment:
What is the idea here ? As you are in a function that has "@login_required" -- so this will always be true ?

Reply:
The reason for adding the 'has_permission' variable is to make sure that the 'user-tools menu' shows up in the base.html template when a user is logged in. Without it being set to True, the menu wouldn't appear. Another option could be directly setting 'has_permission': True, which would work too.